cf_units/__init__.py

```diff
@@ -1674,7 +1671,7 @@ def change_calendar(self, calendar):
             >>> u.change_calendar('standard')
             Unit('days since 1499-12-23T00:00:00', calendar='standard')
 
-        """
+        """  # NOQA E501
```

Comment:
This looks wrong. Can it be removed?

Reply:
Confirmed.  If I take that out I get line 1670 is too long.